pom.xml

```diff
@@ -68,7 +68,7 @@
     <changelist>-SNAPSHOT</changelist>
     <java.level>8</java.level>
     <jenkins.version>2.222.4</jenkins.version>
-    <jackson.version>2.12.4</jackson.version>
+    <jackson.version>2.13.0</jackson.version>
```

Comment:
should also change `revision` to keep in step  (not sure why we do not set that itself to `$jackon.version` but likely it does not work when it comes to release time.